README.md

```diff
@@ -97,16 +97,21 @@ For advanced users, a hidden option `--logConfig <filename.ini>` can be used to
 
 # Release History
 
-## 1.4.0 (April 6, 2019)
+## 1.4.0 (April 12, 2019)
```

Comment:
No version change since Pawel said he will piggy back off changes and bump the version.

Reply:
Oh.  I see.  1.4.0 isn't released yet.  Got it.

I think I'd find it clearer not to have a date there yet for un-released versions.  But no need to change that now.

Reply:
@bwbeach that's a very good point